unittests/testCompositeResourceRetriever.cpp

```diff
@@ -46,7 +47,7 @@ using dart::utils::CompositeResourceRetriever;
 TEST(CompositeResourceRetriever, exists_NothingRegistered_ReturnsFalse)
 {
   CompositeResourceRetriever retriever;
-  EXPECT_FALSE(retriever.exists("package://test/foo"));
+  EXPECT_FALSE(retriever.exists(Uri("package://test/foo")));
```

Comment:
I would prefer to call `fromString` here, instead of using the constructor that calls `fromStringOrPath`, because we know this is a URI. This applies to all of the assertions in this file.